csrankings-g.csv

```diff
@@ -643,7 +643,7 @@ Gregoris Mentzas,NTUA,http://imu.ntua.gr/users/gmentzas,WdybrQYAAAAJ
 Gregory B. White,University of Texas at San Antonio,http://www.cs.utsa.edu/facstaff,yT6jxa8AAAAJ
 Gregory Bodwin,University of Michigan,https://bodwin.engin.umich.edu,NWpUSd4AAAAJ
 Gregory Butler,Concordia University,http://www.cs.concordia.ca/~gregb,K2vSwvQAAAAJ
-Gregory D. Abowd,Georgia Institute of Technology,http://ubicomp.cc.gatech.edu/gregory-d-abowd,gb8sbdcAAAAJ
+Gregory D. Abowd,Northeastern University,https://coe.northeastern.edu/people/abowd-gregory/,gb8sbdcAAAAJ

```

Comment:
Please explain how they meet the inclusion criteria (not in CS).

Reply:
Gregory has a Courtesy Appointment in the Khoury College of Computer Sciences, with PhD student advising privileges. Please see https://www.khoury.northeastern.edu/people/gregory-abowd/